### PR TITLE
feat: Incident list - add search bar

### LINKED ISSUE #1046: Incident list - add search bar

### PR DESCRIPTION
**Pull request by gitstart**
Closes #1046

[Demo](https://www.loom.com/share/d63a80220c82446bb257a226ef83c48e)

---
This code was written and reviewed by GitStart Community. Growing great engineers, one PR at a time.

### CODE REVIEW

**Comment by gitstart** [comment]
@mainawycliffe This PR is ready for review Cc: @moshloop

File: src/api/services/incident.ts [hunk 1]
```diff
@@ -127,12 +127,17 @@ export const getIncidentsWithParams = async (
     : `hypotheses(*,created_by(${AVATAR_INFO}),evidences(id,evidence,type,component_id))`;
 
   const responder = `responders(created_by(${AVATAR_INFO}))`;
+  const { search = "" } = params!;
+  const searchStr = search
+    ? `&or=(title.ilike.*${search}*,status.ilike.*${search}*,severity.ilike.*${search}*,type.ilike.*${search}*)`
```
Comment (moshloop) [inline src/api/services/incident.ts:132]: This should only search for title

Comment (gitstart) [reply src/api/services/incident.ts]: This has been resolved @moshloop, please check again.

**Comment by moshloop** [comment]
@usmailaabdoul - please resign the CLA.

**Review by moshloop — APPROVED** [review]

**Pull request merged**